functions/function_usage_examples/state_preparation/.ipynb_checkpoints/prepare_ghz_state_example-checkpoint.ipynb

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
remove this file and directory

Reply:
I dont know where it came from - is it ok now?